multimodal/src/autogluon/multimodal/data/process_image.py

```diff
@@ -102,8 +106,14 @@ def __init__(
         self.mean = None
         self.std = None
 
+        if self.prefix == "mmdet_image":
```

Comment:
Consider registering "mmdet_image" in constants.py.

Reply:
Done